docs/features.md

```diff
@@ -0,0 +1,100 @@
+---
+id: features
+title: Features
+---
+
+Smart Data Lake Builder is still under heavy development so new features are added all the time.
```

Comment:
I know it will be an absolute pain to keep it up to date, but as a reader who wants to decide if I want to use SDL or not, I would very much like to know what is planned and what is already available. Else I remain very sceptical.
Could we maybe just list current features and link to https://github.com/smart-data-lake/smart-data-lake/issues for coming features?

Reply:
Good point, I inserted a note that the roadmap will be published shortly. We actually had this point in the architecture workshop today. We will start creating milestones with assigned features so people can check the planned features (and the priority of these) in the future.